cli_test.txtar

```diff
@@ -23,3 +23,13 @@ stderr 'HTTP/1.1 200 OK'
 stdout 'Xyz: bar blah'
 stdout 'Content-Type: foo/bar'
 stdout 'POST /test-path HTTP/1.1'
+
+# HTTP multi proxy to run (-M flag)
+fortio curl -M '8088 http://fortio-c1-http:8080 http://fortio-c2-http:8080' https://debug.fortio.org/test-path
```

Comment:
these flags only apply to server mode

Reply:
@ldemailly I try to write test for server mode, but i find a problem, that the process of server mode can't be stop and test go to endless loop. 
As i see in testscript [doc](https://arc.net/l/quote/ofdmazkc), using exec with & token terminate command in background process, but it doesn't seem to be truth, because using kill command doesn't help.
So, i have reverted commit with test and checked flags manually - it works same.

Reply:
server tests exist in webtest.sh (predating txtar which does not do great with multiple processes). Let me see I can do a manual test or add one there